### ISSUE TITLE
LinkIt incomplete?

### ISSUE THREAD

**Issue by kevinsmith** [engram-design/FeedMe-Helpers#3]
Is the LinkIt helper missing support for importing anything other than custom URLs, or am I missing something? It appears that way looking at the code, but I just wanted to be sure before banging my head against the wall trying to get it to work.

**Comment by engram-design** [comment]
Thats correct for the time-being - we needed this for a client very quickly, so its halfway there! I'll keep you posted.
